### PR TITLE
added is-case-allcator guard

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6686

### Change description ###
Added is Case Work Allocator guard

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by maliumer — APPROVED** [review]

File: src/app/shared/guards/is-case-allocator.guard.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import * as fromActions from '../../store';
```
Comment (DavidJayakumar) [inline src/app/shared/guards/is-case-allocator.guard.ts:4]: Sort imports alphabetically, Otherwise happy to approve

Comment (MateiHMCTS) [reply src/app/shared/guards/is-case-allocator.guard.ts:4]: seems right to me

**Review by DavidJayakumar — APPROVED** [review]
Look fine

**Review by udaydenduluri33 — APPROVED** [review]
@MateiHMCTS thanks for the changes

**Pull request merged**